examples/with-next/pages/index.tsx

```diff
@@ -198,8 +198,9 @@ export default class Index extends PureComponent {
     } else {
       this.initialize()
       this.setState({ activeDemo: 'keyword', keyword: { status: 'Calibrating...' } })
-      const speechConfig = {
-        ...DefaultSpeechConfig,
+
+      await startPipeline({
+        ...(PipelineProfiles.Keyword as PipelineProfile),
```

Comment:
I think instead of this, we could add a `profile` property to the options and set the stages internally, and then remove that profile property.

Reply:
One of the points of removing `DefaultSpeechConfig` is to get away from asking the user to do any destructuring in the config.